site/content/docs/tutorials/supervisor-without-concierge-demo.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Learn to use the Pinniped Supervisor without the Concierge
+description: See how the Pinniped Supervisor can work directly with the Kube API server to provide authentication to Kubernetes clusters.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Supervisor without Concierge
+    parent: tutorials
+    weight: 200
+---
+
+## Overview
+
+This tutorial shows how to use the Pinniped Supervisor and Pinniped command-line tool to provide provided federated identity
+with a single sign-on user experience on many Kubernetes clusters, without using the Pinniped Concierge.
+If you would like to learn how to use the Pinniped Supervisor and Concierge together,
+please instead see this other tutorial:
+- [Concierge with Supervisor: a complete example of every step, demonstrated using GKE clusters]({{< ref "concierge-and-supervisor-demo" >}})
+
+The Kubernetes API server can be configured to trust an OIDC identity provider to provide authentication
+for the cluster. This is done by setting the `--oidc-*` command-line flags of the `kube-apiserver` command-line tool inside
+the Pod spec of the Kubernetes API server Pods. The details of these command-line flags are described in the
+[Kubernetes kube-apiserver documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
```

Comment:
We could also link the OIDC-specific docs at https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens